README.md

```diff
@@ -15,6 +15,15 @@ group :development, :test do
 end
 ```
 
+Want to run against the `master` branch? You'll need to include the dependent
+RSpec repos as well. Add the following to your `Gemfile`:
+
+```ruby
+%w[rspec-rails rspec-core rspec-expectations rspec-mocks rspec-support].each do |lib|
```

Comment:
Sure I can update. I didn't even realize they were in alphabetical order. :smile: